Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -228,19 +228,25 @@ void SVGAnimationElement::animationAttributeChanged()
     setInactive();
 }
 
-float SVGAnimationElement::getStartTime() const
+ExceptionOr<float> SVGAnimationElement::getStartTime() const
 {
-    return narrowPrecisionToFloat(intervalBegin().value());
+    SMILTime startTime = intervalBegin();
```

Comment:
I think there is no need to create a new name:
```
auto intervalBegin = this->intervalBegin();
```

Reply:
@shallawa - thanks! done now.